src/bodywork/cli/workflow_jobs.py

```diff
@@ -72,8 +73,7 @@ def delete_workflow_job_in_namespace(namespace: str, job_name: str) -> None:
     print(f"workflow job={job_name} deleted from namespace={namespace}")
 
 
-def create_workflow_cronjob_in_namespace(
-    namespace: str,
+def create_workflow_cronjob(
     schedule: str,
     project_name: str,
```

Comment:
We pass `--name` into this field, which confused the hell out of me. I take full responsibility, but would you mind tidying-up the naming and docstring?

Reply:
This was bugging me too but I didn't want to disrupt the order of things as it's used in a few places.

Reply:
I'll change it in the other places too.

Reply:
Brilliant - thank you!